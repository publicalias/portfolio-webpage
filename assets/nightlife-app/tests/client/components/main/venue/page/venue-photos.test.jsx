"use strict";

//local imports

const VenuePhotos = require("../../../../../../scripts/client/components/main/venue/page/venue-photos");

const { testWrapper } = require("../../../../test-helpers");

//global imports

const { initTestSnapshot, reactTests } = require("redux/tests/react-tests");

//utilities

const { testShallow } = testWrapper(VenuePhotos);

const testSnapshot = initTestSnapshot(testShallow);

//setup

beforeAll(reactTests.setup);
beforeEach(reactTests.inject(VenuePhotos));

//venue photos

test("venue photos should match snapshot", () => testSnapshot());
