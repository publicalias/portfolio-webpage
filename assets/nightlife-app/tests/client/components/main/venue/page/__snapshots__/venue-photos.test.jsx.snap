// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`venue photos should match snapshot 1`] = `""`;
